inc/namespace.php

```diff
@@ -454,6 +420,22 @@ function load_object_cache_redis() {
 	wp_cache_init();
 }
 
+/**
+ * Configure Afterburner's Redis connection for the object cache.
+ */
+function load_object_cache_afterburner() {
+	if ( defined( 'REDIS_AUTH' ) && defined( 'REDIS_HOST' ) && defined( 'REDIS_PORT' ) ) {
+		$secure = defined( 'REDIS_SECURE' ) ? REDIS_SECURE : true;
+		ini_set( 'afterburner.redis_server_info', sprintf( 'redis%s://:%s@%s:%d', $secure ? 's' : '', REDIS_AUTH, REDIS_HOST, REDIS_PORT ) );
```

Comment:
How safe is changing this mid-request?

Reply:
Basically this needs to be set before `wp_cache_init()` is called. If `wp_cache_init()` is called before a valid redis server is provided, it will also re-try with the new details the next time wp_cache_init() is called so it should be safe 👍

Reply:
Ok, either way we set this setting (and others) in docker-entrypoint when running php-fpm so I don't think it's necessary to set again here.

If we do want to support changing the connection string like that, I think we should detect that change in wp_cache_init and explicitly close the previous main thread and pubsub connections, otherwise setting the connection string is a shot in the dark, which will only work if/when the previous connection fails.

Reply:
Ah cool, if you are setting already via `php -fpm -d` (or ini) then that's fine, we can remove.